src/metrics.ts

```diff
@@ -355,6 +355,12 @@ export function getMetrics(
       labelNames: ['topic']
     }),
 
+    duplicateMsgIgnored: register.gauge<{ topic: TopicLabel }>({
+      name: 'gossisub_duplicate_msg_ignored_total',
+      help: 'Total count of duplicate message ignored by topic',
```

Comment:
Need to also mention that this is about the `publish` api, change the name accordingly